### PR TITLE
Byom

### PR DESCRIPTION
**Pull request by ssiegel95**
* adds support for TSFM_MODEL_DIR values in the form `DIR1:DIR2:DIR3`
* resolution rules is that that first that is found to be a valid model_id path will be used
* checks include:
   1. a model_path exists
   2. a model_path is readable by the current process

### CODE REVIEW

File: services/inference/tsfminference/dirutil.py [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# THIS FILE IS COPIED FROM THE BOILERPLATE DIRECTORY,
```
Comment (wgifford) [inline services/inference/tsfminference/dirutil.py:1]: should this be copied from boilerplate?

**Review by wgifford — APPROVED** [review]
Looks good, but please see my comment.

File: services/inference/tsfminference/dirutil.py (hunk 1 above)
Comment (ssiegel95) [reply services/inference/tsfminference/dirutil.py:1]: I think it's probably going to have use in finetuning as well assuming we allow the finetuning of any model that a user might pre-train. If we are going to restrict finetuning to only base models we release then we can move it out of boilerplate.

Comment (wgifford) [reply services/inference/tsfminference/dirutil.py:1]: Yes, but does this particular file need to be tracked in git?

Comment (ssiegel95) [reply services/inference/tsfminference/dirutil.py:1]: Oh, sorry now I understand. No, that's a mistake. I'll remove it, thanks.

**Pull request merged**